api/lambdas/search-by-keywords.js

```diff
@@ -330,6 +330,7 @@ function queryFields() {
     'dc_coverage_spatial',
     'dc_description_ecv',
     'dc_description_ebv',
+    'obps_endorsementExternal_externalEndorsedBy',
```

Comment:
Technically this wasn't needed to support search from the UI but this way endorsed by is included in the All Fields search.